satpy/hrit_msg_tutorial.ipynb

```diff
@@ -32,19 +32,21 @@
     "from satpy.resample import get_area_def\n",
```

Comment:
<p>While you're at it, do you want to change <code>natural</code>  to <code>natural_color</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>